flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
what's the issue with not working on aarch64?

Reply:
I ran into some weird issues, none of them are unsolvable of course, but given that I can't test arm64 locally, I didn't want to hold this up for the majority of users on something that could be finished in a PR later on.

Reply:
I'd encourage Endless to get you an appropriate aarch64 device ;-) half the point of these 'built from source' versions of apps with big name proprietary versions is to be able to support architectures that they don't want to.